responseobjects/elastic_request_builder.py

```diff
@@ -233,19 +236,38 @@ def apply_paging(es_search, pagination):
         Applies the pagination to the ES Search object
         :param es_search: The ES Search object
         :param pagination: Dictionary with raw entries from the GET Request.
-        It has: 'from', 'size', 'sort', 'order'
+        It has: 'size', 'sort', 'order', and one of 'search_after', 'search_before', or 'from'.
         :return: An ES Search object where pagination has been applied
         """
         # Extract the fields for readability (and slight manipulation)
-        _from = pagination['from'] - 1
-        _to = pagination['size'] + _from
-        _sort = '{}.keyword'.format(pagination['sort'])
+
+        _sort = pagination['sort']
         _order = pagination['order']
         # Apply order
-        es_search = es_search.sort({_sort: {"order": _order}})
-        # Apply paging
-        # TODO: We need to handle pagination for more than 10,000 entries
-        es_search = es_search[_from:_to]
+        if 'from' in pagination:
```

Comment:
@jvlasblom if a user sends `from` and `size` within the request, but `from + size` is > `SEARCH_AFTER_THRESHOLD`, wouldn't this break ElasticSearch?

Reply:
Thanks, this was an oversight. I've added a check to return a BAD REQUEST if from+size > SEARCH_AFTER_THRESHOLD

Reply:
Great, thanks!